### ISSUE TITLE
Theme window background showing when opening share action

### ISSUE THREAD

**Issue by barnhill** [urbanairship/android-library#85]
Android application theme contains a background drawable.  A notification share action is tapped when on the home screen (launcher on the phone).  The CoreActivity is shown with a theme of the applications theme which shows the background (in our case a white background) till the share action picker shows up.  Then the CoreActivity goes away and I am returned to the launcher screen, with the action picker still showing.

To circumvent this in the interim:
```
<activity android:name="com.urbanairship.CoreActivity" android:theme="@android:style/Theme.Translucent.NoTitleBar"/>
```
was added to the androidmanifest.xml.

Is there a reason the CoreActivity doesnt have a theme of translucent with no titlebar by default if it has no UI?

Ill help provide any information I can to get this resolved up to an including a pull request.

**Comment by rlepinski** [comment]
Your workaround is fine, we will fix it in the next release. Thanks for reporting this issue.

Not sure if we want `"@android:style/Theme.NoDisplay"` or `"@android:style/Theme.Translucent.NoTitleBar"` , but we will figure it out.

**Comment by barnhill** [comment]
awesome!! Thanks for responding so quickly!!

**Comment by rlepinski** [comment]
Fixed in 9.3.1

**Comment by barnhill** [comment]
Thanks @rlepinski !!  That was quick!
